plugin/tools.py

```diff
@@ -723,12 +724,12 @@ def _get_regular_clang_version_str(cls, output_text):
 
     @classmethod
     def _get_apple_clang_version_str(cls, output_text):
-        version_regex = re.compile(r"\d\.\d\.*\d*")
+        version_regex = re.compile(r"\d*\.\d*\.\d*")
```

Comment:
Same as above.

Reply:
yeah, + is better than *.
Also needs something like `osx_version = '.'.join(version_str.split('.')[:2])` to get first two numbers.